tasks/task612_yorubabbc_classification.json

```diff
@@ -0,0 +1,11499 @@
+{
+    "Contributors": [
+        "Ishan Purohit"
+    ],
+    "Input_language": "Yoruba",
+    "Output_language": "English",
+    "Instruction_language": "English",
+    "Source": [
+        "Yoruba BBC Topics (Dataset - https://huggingface.co/datasets/yoruba_bbc_topics) (Paper - https://aclanthology.org/2020.emnlp-main.204/)"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Definition": "In this task, you're given a news article headlines from Yoruba from BBC Yoruba and your task is to classify each article into the following categories:  Africa, Entertainment, Health, Nigeria, Politics, Sport, or World.",
```

Comment:
clarify whether an article can be mapped to >1 labels?

Reply:
> clarify whether an article can be mapped to >1 labels?

Maybe it could but in the dataset, only one label is given to all the Instances.